src/components/dropdown/index.js

```diff
@@ -0,0 +1,191 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import { motion } from 'framer-motion';
+import { spring } from 'popmotion';
+
+const Dropdown = ({ items }) => {
+  const [clicked, setClicked] = useState(false);
+  const [title, setTitle] = useState('');
+  const [titleHovered, setTitleHovered] = useState(false);
+  const [curIndex, setCurIndex] = useState(-1);
+
+  useEffect(() => {
+    document.addEventListener('keypress', onKeypress);
+    document.addEventListener('keydown', onKeydown);
+    return () => {
+      document.removeEventListener('keypress', onKeypress);
+      document.removeEventListener('keydown', onKeydown);
+    };
+  }, [curIndex, titleHovered, clicked]);
+
+  const onKeypress = e => {
+    if (e.keyCode === 13) {
+      if (clicked) {
+        setTitle(items[curIndex]);
+        setClicked(false);
+      }
+      if (titleHovered) {
+        setClicked(!clicked);
+      }
+    }
+  };
+
+  const onKeydown = e => {
+    if (e.keyCode === 40 || e.keyCode === 9) {
+      if (curIndex + 1 < items.length) setCurIndex(curIndex + 1);
+      else setCurIndex(0);
+    } else if (e.keyCode === 38) {
+      if (curIndex - 1 > -1) setCurIndex(curIndex - 1);
+      else setCurIndex(items.length - 1);
+    }
+  };
+
+  const options = items.map((item, index) => {
+    return (
+      <Option
+        key={`option-${index+1}`}
+        animate={
+          curIndex === index
+            ? { backgroundColor: '#f0f0f0' }
+            : { backgroundColor: '#ffffff' }
+        }
+        whileTap={{ backgroundColor: '#e7e7e7' }}
+        onHoverStart={() => {
+          setCurIndex(index);
+        }}
+        onHoverEnd={() => {
+          setCurIndex(-1);
+        }}
+        noBorder={index === items.length - 1}
+        onClick={() => {
+          setClicked(false);
+          setTitle(item);
+        }}
+      >
+        {item}
+      </Option>
+    );
+  });
+
+  return (
+    <DropdownContainer>
+      <TitleContainer
+        onClick={() => {
+          setClicked(!clicked);
+        }}
+        onHoverStart={() => {
+          setTitleHovered(true);
+        }}
+        onHoverEnd={() => {
+          setTitleHovered(false);
+        }}
+      >
+        <Title>{title}</Title>
+        <ArrowSvgContainer
```

Comment:
Also here maybe use an html symbol instead of svg, but again, matter of preference!

Reply:
Same reasoning as my above comment